devops/vaults.json

```diff
@@ -1,55 +0,0 @@
-[
```

Comment:
Don't we still need this file to build the env variables from 1Password?

Reply:
no, don't need it in UI projects.

Reply:
OK. But you'll have to explain to me (or team) how it works now :)